docroot/modules/custom/yukon_w3_custom/css/yukon_w3_custom.css

```diff
@@ -0,0 +1,58 @@
+ul.tabs--secondary.pagination.pagination-sm {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+ul.tabs--secondary.pagination.pagination-sm li a {
+    position: relative;
+    float: left;
+    padding: 6px 12px;
+    line-height: 1.4375;
+    text-decoration: none;
+    color: #335075;
+    background-color: #eaebed;
```

Comment:
Always use variables for colors, please. It will be much easier to maintain it in the future

Reply:
We have added the variable file as mentioned.